pkg/virt-launcher/virtwrap/converter/vcpu/vcpu.go

```diff
@@ -469,6 +469,11 @@ func AdjustDomainForTopologyAndCPUSet(domain *api.Domain, vmi *v12.VirtualMachin
 			log.Log.Reason(err).Error("failed to calculate passed through NUMA topology.")
 			return err
 		}
+	} else if isNumaMemoryMode(vmi) {
```

Comment:
I'm a bit puzzled. What does it mean if we're setting `domain.NUMATune` without creating `domain.CPU.NUMA.Cells`?

Reply:
It means that we are not exposing NUMA topology into the VM but we want to impact how memory is allocated when we are using resources from multiple NUMA nodes. It can help with memory fragmentation when VM consumes memory from more than one topology zone and uses `interleave` mode. For the sake of completeness I have decided to expose all options that are available in libvirt.